docs/source/api/deliveryservice_server_capabilities.rst

```diff
@@ -0,0 +1,222 @@
+	| xmlID             | no       | Filter associated Server Capabilities by Delivery Service xml ID                                              |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+	| serverCapability  | no       | Filter associated Server Capabilities by Server Capability                                                    |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+	| orderby           | no       | Choose the ordering of the results - must be the name of one of the fields of the objects in the ``response`` |
+	|                   |          | array                                                                                                         |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+	| sortOrder         | no       | Changes the order of sorting. Either ascending (default or "asc") or descending ("desc")                      |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+	| limit             | no       | Choose the maximum number of results to return                                                                |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+	| offset            | no       | The number of results to skip before beginning to return results. Must use in conjunction with limit.         |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+	| page              | no       | Return the n\ :sup:`th` page of results, where "n" is the value of this parameter, pages are ``limit`` long   |
+	|                   |          | and the first page is 1. If ``offset`` was defined, this query parameter has no effect. ``limit`` must be     |
+	|                   |          | defined to make use of ``page``.                                                                              |
+	+-------------------+----------+---------------------------------------------------------------------------------------------------------------+
+
+.. code-block:: http
+	:caption: Request Example
+
+	GET /api/1.4/deliveryservice_server_capabilities HTTP/1.1
+	Host: trafficops.infra.ciab.test
+	User-Agent: curl/7.47.0
+	Accept: */*
+	Cookie: mojolicious=...
+
+Response Structure
+------------------
+:deliveryServiceID: The Delivery Service's ID
+:xmlID:             The Delivery Service's text-based unique identifier
```

Comment:
Please link to the relevant section of the Delivery Service overview (label is `ds-xmlid`)